app/src/main/java/com/github/factotum_sdp/factotum/ui/picture/PictureFragment.kt

```diff
@@ -74,7 +108,7 @@ class PictureFragment(clientID: String) : Fragment() {
         if (isGranted) {
             openCamera()
         } else {
-            findNavController().popBackStack()
+            findNavController().navigateUp()
```

Comment:
Just removing that call line, and doing nothing in the else, let the device going back where the Intent have been launched, without having an Exception.